packages/circuit-ui/components/ProgressBar/ProgressBar.mdx

```diff
@@ -7,21 +7,14 @@ import * as Stories from './ProgressBar.stories';
 
 <Status.Stable />
 
-A progress bar is a UI element used to visualise the progression of a task or operation. The visual element is always combined with a label describing a status value either in percentage or fraction format.
+A progress bar is a UI element used to visualise the progression of a task or operation. The visual element is always combined with a label describing a status value, either in percentage or fraction format.
 
 <Story of={Stories.Steps} />
 <Props />
 
-## When to use it
-
-The progress bar should be used to indicate the amount of steps or time left to complete a task. It should not however, be used to indicate the time needed to load a page.
-
 ## Usage guidelines
 
-- **Do** place the status value to the right of the progress bar.
-- **Do** use in moderation, only for tasks, operations or information that may take a long time.
-- **Do not** use an indeterminate value beyond percentage or fraction.
-- **Do not** use a progress bar for every task, operation or information.
+The progress bar should be used to indicate the amount of steps or time left to complete a task. It should not, however, be used to indicate the time needed to load a page. For a loading state, use the [Skeleton component](Components/Skeleton) or the [Spinner component](Components/Spinner).
```

Comment:
The Skeleton component is an internal component (i.e. not publicly exported). Do you want to make it public?

Reply:
🙀 my bad, I thought it was! Maybe we should? It's used in production anyways via the nav components. We can then iterate on docs for it to make it more useful to teams. But looking at some of our apps, it seems like a lot of people are implementing their own kind of skeletons.

Otherwise, I'm also happy to revert a part of the docs changes for now 🙂  What do you think?

Reply:
I didn't make the Skeleton component public when I built the new navigation components because I wasn't sure about its API and design. We should review the component (including a design review with Boris) before we release and promote it.

So yes, I'd revert the docs changes for now. Sorry!